core/templates/pages/exploration-player-page/layout-directives/feedback-popup.component.html

```diff
@@ -55,6 +55,9 @@
   label {
     font-weight: normal;
   }
+  .fa-times {
```

Comment:
```suggestion
  .oppia-feedback-popup-container .fa-times {
```

Reply:
Please ensure this CSS rule is scoped so it doesn't affect other usages of this class elsewhere.

Reply:
Got it! Thanks